rllib/models/torch/encoders/vector.py

```diff
@@ -0,0 +1,129 @@
+from typing import TYPE_CHECKING, List
+
+import torch
+from torch import nn
+
+from ray.rllib.models.specs.specs_dict import ModelSpec
+from ray.rllib.models.torch.model import TorchModel
+from ray.rllib.models.utils import get_activation_fn
+from ray.rllib.utils.nested_dict import NestedDict
+
+if TYPE_CHECKING:
+    from ray.rllib.models.configs.encoder import VectorEncoderConfig
+
+
+def input_to_output_spec(
```

Comment:
We should move this to a non-torch specific utils.py file.
Maybe: `rllib/models/encoder_utils.py`